test/fixtures/amp-bind-integrations.html

```diff
@@ -136,5 +134,11 @@
   <amp-list id='list' src="https://www.google.com/unbound.json" [src]=listSrc>
   </amp-list>
 
+  <p>AMP-STATE</p>
+  <button id="ampStateSrcButton" on="tap:AMP.setState({ampState: {ampStateSrc: 'https://www.google.com/bind/second/source'}})">Change amp-state</button>
```

Comment:
Oh I meant the nested JSON key actually:
```
AMP.setState({ampState: {mySrc: 'https://...'}})
```
Since `ampState.ampStateSrc` is a bit harder to read than `ampState.mySrc`.